metatron-app/metatron-discovery
# #3427 Hot fix for #3411 Remove variable unUsedVirtualColumnName.

It also includes improvements in treating variable containing space and checking user-defined function containing aggregation functions. It also includes a patch for #3413 

### Description
https://github.com/metatron-app/metatron-discovery/issues/3427
https://github.com/metatron-app/metatron-discovery/issues/3413

**Related Issue** : 
Additionally, the logic for determining whether a field includes aggregation has been improved for this issue and treating variable containing space like 'total income'.

### How Has This Been Tested?
See the issue
https://github.com/metatron-app/metatron-discovery/issues/3427

Additionally, we can test following.
1. Create user-defined field 'total sales' : SUMOF([Sales])
2. Create user-defined field 'total profit' : SUMOF([Profit])
3. Create user-defined field 'profit ratio' : [user_defined.total profit] / [user_defined.total sales] * 100
4. Check discovery treats the user-defined field 'profit ratio' as an aggregation function.

For #3413 
1. Create user-defined field 'measure_1' : COUNTD(IF("=[ORDERID] != '', [ORDERID], '')) or COUNTD(CONCAT([ORDERID], [REGION]))
2. Check discovery can draw chart.

#### Need additional checks?
We need to improve for treating dimensions containing aggregation.

### Types of changes



### Checklist:

- [] My change requires a change to the documentation.

### Additional Context
